tests/app/service/test_rest.py

```diff
@@ -1475,6 +1467,9 @@ def test_get_detailed_services_for_date_range(
     assert data[0]['statistics'][LETTER_TYPE] == {'delivered': 0, 'failed': 0, 'requested': 0}
 
     # Teardown
+    notify_db_session.session.execute(
+        delete(FactNotificationStatus).where(FactNotificationStatus.service_id == template.service.id)
+    )
```

Comment:
This should be in a try/finally, so it always cleans up. The notification deletion below should be removed because you made a sample_notification.

Reply:
Updated.